pkg/app/AppService.go

```diff
@@ -2176,17 +2176,27 @@ func (impl *AppServiceImpl) createHelmAppForCdPipeline(overrideRequest *bean.Val
 				ReleaseIdentifier: releaseIdentifier,
 			}
 			helmResponse, err := impl.helmAppClient.InstallReleaseWithCustomChart(ctx, helmInstallRequest)
+
+			//IMP: update cd pipeline to mark deployment app created, even if helm install fails
```

Comment:
why we have to mark this true if app does not created

Reply:
discussed